frontend/src/components/input-fields/form-divider/form-divider.tsx

```diff
@@ -0,0 +1,33 @@
+import { styled } from "styled-components";
+
+import { FormDividerProps } from "./form-divider.props";
+import { color, spacing } from "../../../theme";
+
+const DividerContainer = styled.div`
+  padding-top: ${spacing("medium")};
+  width: 100%;
+  display: flex;
+  align-items: center;
+`;
+
+const Line = styled.hr`
+  flex-grow: 1;
+  margin: 0px ${spacing("small")};
+  min-width: 10px;
+
+  height: 1px;
+  background-color: ${color("text")};
+  border-width: 0;
+`;
+
+const Label = styled.h3``;
+
+export const FormDivider: React.FC<FormDividerProps> = ({ label = "" }) => {
+  return (
+    <DividerContainer>
+      <Line />
+      <Label>{label}</Label>
+      <Line />
```

Comment:
Ich fände es cool, dass die Line geschlossen wird, wenn es kein Label gibt.

```suggestion
      {label && <Label>{label}</Label>}
      {label && <Line />}
```